src/wasmtime/crates/lind-common/src/lib.rs

```diff
@@ -69,17 +69,26 @@ impl LindCommonCtx {
             // exit syscall
             EXIT_SYSCALL => wasmtime_lind_multi_process::exit_syscall(caller, arg1 as i32),
```

Comment:
what I mean by unify is making things like exit() here, as well as clone/exec go through make_syscall as well